### DISCUSSION TITLE
Questions / Requests / Balance suggestions

### DISCUSSION THREAD

**Discussion by ATLASLORD** [ATLASLORD/Thorosmen#7] [Forum Thread Thorosmen]
Have any questions, suggestions, or requests about Thorosmen?
Feel free to leave a comment here.
Your Commander's input is appreciated!

**Comment by Walter-Hulsebos** [comment]
Does it work well combined with Legion?

**Reply by ATLASLORD** [reply]
They should work fine with each other but there may be some compatibility issues like shields not shooting down modded units ammos from the other faction.
